pkg/download/protocol.go

```diff
@@ -69,7 +70,9 @@ type protocol struct {
 }
 
 func (p *protocol) List(ctx context.Context, mod string) ([]string, error) {
-	const op errors.Op = "protocol.List"
+	const op errors.Op = "storage.protocol.List"
```

Comment:
this should be `download.Protocol.List`. I am trying to adhere to the convention `package.struct.func` to avoid naming conflicts and help visualize span better.